### PR TITLE
Transfer empty message#190

### PR DESCRIPTION
**Pull request by fboucquez**
Fix of https://github.com/nemtech/nem2-sdk-java/issues/190

### CODE REVIEW

File: sdk-core/src/main/java/io/nem/sdk/model/transaction/TransactionState.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.nem.sdk.model.transaction;
+
+/**
+ * State of the transaction.
+ */
+public enum TransactionState {
```
Comment (Wayonb) [inline sdk-core/src/main/java/io/nem/sdk/model/transaction/TransactionState.java]: Missing the "partial" state for bonded tx.

Comment (fboucquez) [reply sdk-core/src/main/java/io/nem/sdk/model/transaction/TransactionState.java]: @Wayonb , can you clarify?

Comment (Wayonb) [reply sdk-core/src/main/java/io/nem/sdk/model/transaction/TransactionState.java]: Sorry. for an aggregate bonded transaction when it first gets submitted, it goes to the pt cache(partial transaction queue) and wait for cosigners to sign the tx.  When the transaction is in this queue the state should be partial.e.g. Below is the status of a transaction from rest gateway when in that queue.

{"group":"partial","code":"Success","hash":"F7A99B130CFD5641B71CCABC538CE893164AF84229076541A9C8A47F1BF90677","deadline":"5520432181","height":"0"}

Comment (fboucquez) [reply sdk-core/src/main/java/io/nem/sdk/model/transaction/TransactionState.java]: Hi @Wayonb , great catch.

@dgarcia360, I think we are missing the 'partial' value here:

https://github.com/nemtech/nem2-openapi/blob/master/spec/core/transaction/schemas/TransactionStateTypeEnum.yml

Once we have a new version I can add that value to the java enum value.

Comment (fboucquez) [reply sdk-core/src/main/java/io/nem/sdk/model/transaction/TransactionState.java]: I'm creating a new issue as it's unrelated to this PR and I'm blocked for the Open API spec. Thanks for reporting it. @Wayonb could you review the rest of the PR?

**Pull request merged**